src/main/java/net/sf/jabref/model/metadata/MetaData.java

```diff
@@ -147,6 +151,23 @@ public boolean isProtected() {
         return isProtected;
     }
 
+    public Optional<ContentSelectors> getContentSelectors() {
+        return Optional.ofNullable(contentSelectors);
```

Comment:
I think `contentSelectors` is never null, but of course may be empty. Thus the `Optional` part should be removed.